From 1b484c5b288d03d8551e14fb9866074e2763f7c5 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 17 Nov 2020 19:12:31 +0000
Subject: [PATCH 1035/1077] MANTIS 0059400 Need MIB for BIOMETRICS component -
 Add subtype to platformStatus.idl and set subtype in component code

platform.idl V5.3

add LoggingClass::LC_SERVICE
add LoggingComponentData::subType


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1607 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index 325849ef..9494c95e 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -27,7 +27,7 @@
 
 module PlatformManagement
 {
-    const string interface_version = "5.2";
+    const string interface_version = "5.3";
 
     /** Platform interface for the device components. Via this interface the device
         components communicate with the platform manager internally. */
@@ -52,7 +52,8 @@ module PlatformManagement
       LC_UNDEFINED,
       LC_PLF,
       LC_INDICATOR,
-      LC_AUDIO
+      LC_AUDIO,
+      LC_SERVICE
     };
 
     /** Data type for text lists. */
@@ -234,9 +235,10 @@ module PlatformManagement
                                 // - use one unitId for LC_APPLICATION, LC_PRINTER, LC_SCANNER, LC_CONVEYOR
                                 //      LC_PRINTER_FEEDER and LC_CONVEYOR_FEEDER
                                 // - LC_CAPTURE and LC_DISPENSER may be responsible for multiple units
-      LoggingClass unitClass;// class of component
+      LoggingClass unitClass;   // class of component
                                 //   LC_BIN* not used in LoggingComponentData
       string moduleName;        // needed for GAC_MODULENAME
+      string subType;           // READER_TYPE for LC_SCANNER
     };
 
     struct LoggingCounterData
-- 
2.41.0.windows.1

